schemas/campaigns/activity.schema.json

```diff
@@ -34,6 +34,65 @@
           "type": "string",
           "description": "Activity title."
         },
+        "xdm:brief": {
+          "title": "Brief",
+          "type": "string",
+          "description": "An HTML brief describing the activity."
+        },
```

Comment:
I put `brief` in the Experience object, but now realize that we're calling this value `description` in the UI. I was going to change it to `description` in Experience, I don't feel strongly about that change, but it feels like it should be consistent.

Reply:
It's good with me. We do use description in Campaign too ;-)